### ISSUE TITLE
Incorrect rendering of path to function definition on hover in SublimeText

### ISSUE THREAD

**Issue by ShrykeWindgrace** [haskell/haskell-language-server#222]
When I hover on a function, the path to its definition is given without path separator characters (screenshot for illustration): 
![image](https://user-images.githubusercontent.com/6421233/88277098-6f48bd00-cce0-11ea-99df-6604abe6a3ed.png)

Config: Win10-1809, hls.exe from the 0.2.1 release (for GHC-8.10.1). SublimeText4 (4079) with the recommended LSP plugin.

**Comment by Anton-Latukha** [comment]
Is it already solved? Maybe #222 solved it?

**Comment by ShrykeWindgrace** [comment]
I'm on vacation without access to my devenv; I will be able to look into current rendering of paths not earlier than mid-January.

**Comment by jneira** [comment]
@ShrykeWindgrace did you get to check if the issue is reproduced? thanks!

**Comment by ShrykeWindgrace** [comment]
@jneira thanks for pinging me about this issue, I completely forgot about it (I migrated to VSCode since then). I will try to take a look (this week, most probably; but certainly not today).

**Comment by ShrykeWindgrace** [comment]
TLDR: the issue is still there.

![image](https://user-images.githubusercontent.com/6421233/153023773-0431ebb4-2141-4d99-bcff-5c73345d01f2.png)

Config: win10, Sublime Text 4 (4126, latest at the time of writing), HLS-1.6.1.0 for ghc-8.10.7. LSP plugin from https://lsp.sublimetext.io/.

If there is an option that would allow me provide more info, I'm willing to try. I am not entirely sure whether this is an HLS issue or the LSP plugin issue... VSCode renders these paths with backslashes; what does HLS send? forward slashes or backslashes?

**Comment by michaelpj** [comment]
Tentatively closing as no reports recently.
